src/main/java/cn/nukkit/Server.java

```diff
@@ -1367,7 +1371,7 @@ public static int getGamemodeFromString(String str) {
             case "v":
                 return Player.SPECTATOR;
         }
-        return -1;
+        return Player.SURVIVAL;
```

Comment:
I don't think this is very good idea because there is gameMode == -1 checks in the code and this will break them